companion/src/firmwares/eeprominterface.h

```diff
@@ -162,7 +164,10 @@ enum Capability {
   HasAntennaChoice,
   HasADCJitterFilter,
   HasTelemetryBaudrate,
-  TopBarZones
+  TopBarZones,
+  Has,
```

Comment:
@elecpower what is the meaning of this capability?

Reply:
Guessing a change of mind...will be gone soon